README.md

```diff
@@ -1,2 +1,14 @@
 # goto-audio
 Goto the tab that is playing audio. Exclusive for Chrome
+
+Steps to install goto-audio
```

Comment:
prefix this with # so that it becomes a header. try putting a horizontal line between the description and this header! Thanks